src/components/settings/IntegrationsManager.tsx

```
@@ -0,0 +1,291 @@
+        e.preventDefault();
+        setIsSubmitting(true);
+        const success = await addIntegration(platform, webhookUrl, channelName);
+        if (success) {
+            setShowForm(false);
+            setWebhookUrl('');
+            setChannelName('');
+        }
+        setIsSubmitting(false);
+    };
+
+    // Initialize preview on mount
```

ESLint

[error] Delete `··`

(prettier/prettier)